storage/sectorstorage/resources.go

```diff
@@ -0,0 +1,135 @@
+package sectorstorage
+
+import (
+	"github.com/filecoin-project/go-sectorbuilder"
+	"github.com/filecoin-project/lotus/storage/sectorstorage/sealtasks"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+)
+
+var FSOverheadSeal = map[sectorbuilder.SectorFileType]int{ // 10x overheads
+	sectorbuilder.FTUnsealed: 10,
+	sectorbuilder.FTSealed:   10,
+	sectorbuilder.FTCache:    70, // TODO: confirm for 32G
+}
+
+var FsOverheadFinalized = map[sectorbuilder.SectorFileType]int{
+	sectorbuilder.FTUnsealed: 10,
+	sectorbuilder.FTSealed:   10,
+	sectorbuilder.FTCache:    2,
+}
+
+type Resources struct {
+	MinMemory uint64 // What Must be in RAM for decent perf
+	MaxMemory uint64 // Mamory required (swap + ram)
+
+	MultiThread bool
+	CanGPU      bool
+
+	BaseMinMemory uint64 // What Must be in RAM for decent perf (shared between threads)
+}
+
+const MaxCachingOverhead = 32 << 30
+
+var ResourceTable = map[sealtasks.TaskType]map[abi.RegisteredProof]Resources{
```

Comment:
Can we make this a configuration file so that tweaking doesn't have to be a recompilation of the binary?

Reply:
There's nothing configurable about this table, it's just the resource use of sealing stages

Reply:
(which is the same no matter what machine you have)